plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,6 @@ Export-Package: com.google.cloud.tools.eclipse.util,
  com.google.cloud.tools.eclipse.util.templates.appengine
 Import-Package: com.google.common.base;version="[20.0.0,21.0.0)",
  com.google.common.io;version="[20.0.0,21.0.0)",
- com.google.common.net;version="[20.0.0,21.0.0)",
```

Comment:
It's `HttpHeaders.USER_AGENT` in `FileDownloader.java:86`: "Access restriction: The field 'HttpHeaders.USER_AGENT' is not API". Perhaps the reason you don't have the compile error is that this type of error is to set to a warning in your Eclipse. (It compiles anyway as can be seen in Travis. Happened before too: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1320.)